projects/cdk/schematics/ng-update/v4/steps/templates/migrate-axes.ts

```diff
@@ -0,0 +1,45 @@
+import type {UpdateRecorder} from '@angular-devkit/schematics';
+import type {DevkitFileSystem} from 'ng-morph';
+
+import {findElementsByTagName} from '../../../../utils/templates/elements';
+import {
+    getTemplateFromTemplateResource,
+    getTemplateOffset,
+} from '../../../../utils/templates/template-resource';
+import type {TemplateResource} from '../../../interfaces';
+import {removeAttrs} from '../utils/remove-attrs';
+
+const AXES_TAG_NAME = 'tui-axes';
+const AXES_NEW_ATTRIBUTE = 'new';
+
+export function migrateAxes({
+    resource,
+    recorder,
+    fileSystem,
+}: {
+    fileSystem: DevkitFileSystem;
+    recorder: UpdateRecorder;
+    resource: TemplateResource;
+}): void {
+    const template = getTemplateFromTemplateResource(resource, fileSystem);
+    const templateOffset = getTemplateOffset(resource);
+
+    const elements = findElementsByTagName(template, AXES_TAG_NAME);
+
+    elements.forEach(({attrs, sourceCodeLocation}) => {
+        const axesNewAttr = attrs.find((attr) => attr.name === AXES_NEW_ATTRIBUTE);
+
+        if (!sourceCodeLocation) {
+            return;
+        }
+
+        if (axesNewAttr) {
+            removeAttrs([axesNewAttr], sourceCodeLocation, recorder, templateOffset);
```

Comment:
Let's just extend `withTagNames` array of this object:
https://github.com/taiga-family/taiga-ui/blob/78ef961d677fd8056599af2df03cbd1abb33c1e5/projects/cdk/schematics/ng-update/v4/steps/constants/attrs-to-replace.ts#L234-L240

with the following object

```ts
{
    from: {
        attrName: 'new',
        withTagNames: ['tui-progress-circle', 'tui-axes'],
    },
    to: {attrName: ''},
},
```